### PR TITLE
[Release] Merge v1.10.0 release into dev

### PR DESCRIPTION
**Pull request by fernandofloresg**
##### SUMMARY

Merge release changes into dev.

##### ISSUE TYPE

- Release Pull Request
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -78,6 +78,20 @@ releases:
       name: zos_tso_command
       namespace: ''
     release_date: '2022-06-07'
+  1.10.0:
+    changes:
+      release_summary: 'Release Date: ''2024-06-11''
```
Comment (richp405) [inline changelogs/changelog.yaml:83]: Shouldn't this be 2024-06-21?

File: changelogs/fragments/v1.10.0_summary.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+release_summary: |
+  Release Date: '2024-06-11'
```
Comment (richp405) [inline changelogs/fragments/v1.10.0_summary.yml:2]: Again, I think this should be 6-21 for the date.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -4,13 +4,13 @@ ibm.ibm\_zos\_core Release Notes
...
-v1.10.0-beta.1
-==============
+v1.10.0
+=======
 
 Release Summary
 ---------------
 
-Release Date: '2024-05-08'
+Release Date: '2024-06-11'
```
Comment (richp405) [inline CHANGELOG.rst:13]: 6-21

**Review by richp405 — APPROVED** [review]
A few date entries... nothing major.

File: changelogs/changelog.yaml (hunk 1 above)
Comment (fernandofloresg) [reply changelogs/changelog.yaml:83]: well it was created during the release process so is inaccurate

**Comment by ddimatos** [comment]
Connecting the issue so it the PR shows the relationship. #1150

**Pull request merged**